llvm/include/llvm/Analysis/CtxProfAnalysis.h

```diff
@@ -9,23 +9,39 @@
 #ifndef LLVM_ANALYSIS_CTXPROFANALYSIS_H
 #define LLVM_ANALYSIS_CTXPROFANALYSIS_H
 
+#include "llvm/ADT/StringMap.h"
```

Comment:
I didn't see a StringMap being introduced, there is a new DenseMap though?

Reply:
+1